misc.sk

```diff
@@ -75,10 +75,232 @@ plants:
 	chorus fruit¦s = minecraft:chorus_fruit
 	popped chorus fruit¦s = minecraft:chorus_fruit_popped
 
-buckets:
-	[empty] bucket = minecraft:bucket
-	water bucket = minecraft:water_bucket
-	lava bucket = minecraft:lava_bucket
-	milk [bucket] = minecraft:milk_bucket
+old buckets:
+	minecraft version = 1.12.2 or older
+	[empty] bucket¦s = minecraft:bucket
+	water bucket¦s = minecraft:water_bucket
+	lava bucket¦s = minecraft:lava_bucket
+	milk [bucket¦s] = minecraft:milk_bucket
+	any bucket = empty bucket, water bucket, lava bucket, milk
+new buckets:
+	minecraft version = 1.13 or newer
+	tropical fish bucket¦s = minecraft:clownfish_bucket
```

Comment:
It looks to me, [from the wiki](https://minecraft.gamepedia.com/Bucket_of_Fish#ID), that the ID for this is `tropical_fish_bucket`, not `clownfish_bucket`.